src/open_inwoner/utils/ckeditor.py

```diff
@@ -78,7 +78,10 @@ def get_product_rendered_content(product):
                 element.string = ""
                 element.attrs.update(
                     {
-                        "class": "button button--textless button--icon button--icon-before button--primary cta-button",
+                        "class": (
+                            "utrecht-button button--textless button--icon "
```

Comment:
Currently we do not have NLDS button components yet so you can remove this particular button stuff here and revert it to what it was, or else the styling won't work.
I will be changing this one dramatically when I continue work on the Buttons (the NLDS buttons are no longer part of a sprint currently).